### PR TITLE
[Feature] Make homepage banner clickable

### PR DESCRIPTION
**Pull request by midori0507**
- Wrap div in big React-router Link component
- Use 12 col in mobile mode.

### CODE REVIEW

File: app/assets/styles/customization/espoo/customization.scss [hunk 1]
```diff
@@ -111,11 +111,16 @@
     padding-top: 50px;
   }
 
-  .app-HomePage .app-HomePageContent__banner
+  .app-HomePage
     div {
       background-color: $blue-light;
```
Comment (rBoost) [inline app/assets/styles/customization/espoo/customization.scss]: This selector looks a bit scary to me, could we make it more precise than: "all divs under .app-HomePage"?

Comment (midori0507) [reply app/assets/styles/customization/espoo/customization.scss]: agree, but i don't even know if it have some side-effect if we target only 1 div in our fix..

Comment (midori0507) [reply app/assets/styles/customization/espoo/customization.scss]: Ah nvm, i guess it's just selector, will remove without any hurts

**Comment by SamuelSaarentola** [comment]
[Make homepage banners clickable](https://trello.com/c/GmQJSKmy/42-make-homepage-banners-clickable)

**Pull request merged**